CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ### 0.10.6 (Next)
 
 * Your contribution here.
+* [#174](https://github.com/slack-ruby/slack-ruby-bot/pull/174): adds info on http_proxy usage to README - [@phantomdata](https://github.com/phantomdata).
```

Comment:
Danger is complaining because this starts with a lowercase `a` and not an uppercase `A`. We don't however add changelog entries for doc updates, so you can just remove it and let it warn.